js/app.js

```diff
@@ -0,0 +1,145 @@
+document.addEventListener('DOMContentLoaded', function () {
+
+  /* ---------------------------------------------------- */
+  /* ----- SLIDER (SLIDESHOW) IMAGES ----- */
+  /* ---------------------------------------------------- */
+
+  let slideIndex = 0;
```

Comment:
Deklarujesz zmienne w zakresie globalnym `global scope`. Unikaj takich praktyk. Jeśli możesz to deklaruj zmienne wewnątrz swoich funkcji.